.gitignore

```diff
@@ -1,2 +1,3 @@
 .eslintrc
 .gitignore
```

Comment:
you can probably remove `.gitignore` since your `.gitignore` file has already been committed to github